### ISSUE TITLE
Recieve error "Something went wrong..." when upload data from PostgreSQL database

### ISSUE THREAD

**Issue by Arkronus** [amundsen-io/amundsen#955]
I have uploaded information about my postgresql tables. When I try to see Information about tables I see error "Something went wrong... "

## Expected Behavior
I want to see metadata information about tables.

## Current Behavior
I can see that upload was succesfull because information about tables availible in neo4j store. 

I have used docker-amundsen.yml script for deloyment. In amundsenfrontend container logs recieve this error

`2021-03-14T13:11:33+0000.463 [ERROR] v0._get_table_metadata:149 (11:MainThread) - Encountered exception: {'columns': {0: {'col_type': ['Field may not 
be null.']}, 1: {'col_type': ['Field may not be null.']}}}
Traceback (most recent call last):
  File "/app/amundsen_application/api/metadata/v0.py", line 143, in _get_table_metadata
    results_dict['tableData'] = marshall_table_full(table_data_raw)
  File "/app/amundsen_application/api/utils/metadata_utils.py", line 109, in marshall_table_full
    table: Table = schema.load(table_dict).data
  File "/usr/local/lib/python3.7/site-packages/marshmallow/schema.py", line 588, in load
    result, errors = self._do_load(data, many, partial=partial, postprocess=True)
  File "/usr/local/lib/python3.7/site-packages/marshmallow/schema.py", line 711, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'columns': {0: {'col_type': ['Field may not be null.']}, 1: {'col_type': ['Field may not be null.']}}}       
2021-03-14T13:11:33+0000.463 [DEBUG] action_log_callback.on_post_execution:70 (11:MainThread) - Calling callbacks: [<function logging_action_log at 0x7f26506bf8c0>]
2021-03-14T13:11:33+0000.464 [DEBUG] action_log_callback.logging_action_log:85 (11:MainThread) - logging_action_log: ActionLogParams(command='_get_table_metadata', start_epoch_ms=1615727493322, end_epoch_ms=1615727493463, user='test@email.com', host_name='ee9f995f6b9d', pos_args_json='[]', keyword_args_json='{"table_key": "postgres://ioekgftt.public/demo", "index": "0", "source": "search_results"}', output='{"tableData": {}, "msg": "Encountered exception: {\'columns\': {0: {\'col_type\': [\'Field may not be null.\']}, 1: {\'col_type\': [\'Field may not be null.\']}}}", "status_code": 500}', error=None)`

## Steps to Reproduce
1. Deploy amundsen on docker
2. Use sample_postgres_loader.py to upload data

## Your Environment
* Amunsen version used:  amundsen-frontend:3.1.0, amundsen-search:2.4.1, amundsen-metadata:3.3.0
* Data warehouse stores: postgresql
* Deployment (k8s or native): native(docker)

**Comment by feng-tao** [comment]
@allisonsuarez @dkunitsk I thought this is not an issue with the latest version. I think we may want to specify it in the FAQ/installation doc.

**Comment by NiravLangaliya** [comment]
I am also facing same issue and we have already data available into Neo4J with column type , column name , column description is available. so I feel this issue is not related to data coming from PostgreSQL as we have store Hive Metastore data and still we are facing this issue.

**Comment by alldoami** [comment]
I am getting the same error as above ^^. After updating to a newer metadata version (one with Badge support), this started to break. I am on on `latest` for both `metadata` and `frontend-oidc`.

**Comment by feng-tao** [comment]
@alldoami did you upgrade to the latest databuilder version?

**Comment by alldoami** [comment]
@feng-tao where would I specify the latest databuilder version? I don't see it in the `values.yml`

**Comment by feng-tao** [comment]
@alldoami the version you defined in your airflow cluster

**Comment by alldoami** [comment]
ah that worked! Thank you @feng-tao! I'm on the latest for everything btw. Hopefully that will work for others!

**Comment by alldoami** [comment]
@feng-tao also, I found that after updating to `amundsen-databuilder==4.2.1`, I had to add `amundsen-rds>=0.0.3` to my `requirements.txt`, but that broke my scheduler... I reverted that change and it works again. But I still get `No module named 'amundsen_rds'` as an import error.

**Comment by feng-tao** [comment]
I think we add the deps in CI but miss in setup.py (https://github.com/amundsen-io/amundsendatabuilder/pull/462#pullrequestreview-621620935) which will fix it. The new release will be release once pr is merged.

**Comment by feng-tao** [comment]
@alldoami could you use https://pypi.org/project/amundsen-databuilder/4.2.2/ which should fix the issue?

**Comment by alldoami** [comment]
@feng-tao odd... my circle deployment broken when updating it to `4.2.2`... I would just do `amundsen-databuilder==4.2.2`?

**Comment by alldoami** [comment]
@feng-tao updated to `amundsen-databuilder==4.2.2` but still get the same issue 😢  I still get the `No module named `amundsen_rds``

**Comment by feng-tao** [comment]
@alldoami I wonder how do you installed the package? is there any cache for previous installed?

**Comment by alldoami** [comment]
@feng-tao i have this in my Dockerfile: 
```
COPY requirements-dev.txt ./
RUN pip install --no-cache-dir -r requirements-dev.txt
```
Where the packages are listed as a normal `requirements.txt` would be formatted.

**Comment by alldoami** [comment]
I also just ran: 
```
# pip freeze | grep amundsen-databuilder
amundsen-databuilder==4.2.2
```
So the package is updated.

**Comment by feng-tao** [comment]
@alldoami I was confirmed with a different deployment that `amundsen-databuilder==4.2.2` should fix your initial issue. I am not sure what else could cause your current error.

**Comment by alldoami** [comment]
@feng-tao so I shouldn't need to include the `amundsen-rds` package in the `requirements.txt`?

**Comment by feng-tao** [comment]
@alldoami  I see this issue now... your airflow dependency is a static defined one in requirements.txt. Currently Amundsendatabuilder puts the dependency in setup.py (https://github.com/amundsen-io/amundsendatabuilder/blob/master/setup.py#L21) in installed-required which will pulled any transient dependency as well.

I think instead of hand-writing requirements.txt, you need to use https://github.com/jazzband/pip-tools to generate requirements.txt from a requirements.in / setup.py which defined all the must-have dependencies. Then the generated requirements.txt will include the must-have dependencies as well as transient dependencies.

**Comment by feng-tao** [comment]
tldr, `amundsen-rds` is needed but you shouldn't manually updated your requirements.txt

**Comment by cn-dinakarsundar** [comment]
@feng-tao  , We are also facing the same Issue , do i need to drop the data in neo4j and Ingest the data from using latest amundsen-databuilder==4.2.2 or else should i mention this in the front requirments.txt is fine ?
